src/app/data/project-data.ts

```diff
@@ -564,6 +564,32 @@ export const projectsData = [
             "simulator"
         ]
     },
+    {
+        name: "Horizonpedia",
+        description: "App for iOS and Android with Item catalog, fake art comparison, and more!",
+        languages: ["en", "es", "de", "fr", "it", "jp", "ko", "nl", "ru", "zh", "se"],
+        logo: "",
+        logoStyle: "",
+        site: "",
+        android: "https://play.google.com/store/apps/details?id=com.wegner.horizonpedia",
+        ios: "https://testflight.apple.com/join/cYhwMhdD",
+        discord: "https://discord.gg/pQMRkMg",
```

Comment:
Only the unique invite code should be here (`"pQMRkMg"`)

Reply:
Does it really resolve to testflight automatically?